TgeeT/Questioner_v2
# Feature #163279149 login a user

#### What does this PR do?

This PR create an endpoint for users to login to the application
#### Description of Task to be completed?

- Ensure a user can login with details they registered with

#### How should this be manually tested?
How should this be manually tested?

follow the steps below:
1. git clone this repository [here](https://github.com/TgeeT/Questioner_v2.git) 
2. Create a virtual environment and activate it.
3. git checkout to ft-login-user-163279149 branch
4. Install requirements with 'pip install -r requirements.txt'
5. Run the application and with post man, post at the endpoint 'api/v2/user'

### What are the relevant pivotal tracker stories?

https://www.pivotaltracker.com/story/show/163279149

### Screenshots
###### postman
![image](https://user-images.githubusercontent.com/45405036/51519336-bfb43580-1e31-11e9-991b-830438d68546.png)
